client/src/styles/global.scss

```diff
@@ -1,2 +1,31 @@
-@import "~@trussworks/react-uswds/lib/uswds.css";
 @import "~@trussworks/react-uswds/lib/index.css";
+@import "~@trussworks/react-uswds/lib/uswds.css";
+
+.j40FloatLogoLeft {
+  width: 100%;
+  padding-left: 50%;
+}
+
+.j40Footer {
+  font-family: "sans-serif";
+  font-size: 12px;
+  line-height: 24px;
+}
+
+.j40FooterAddressTitle {
+  line-height: 2.5;
+  font-weight: bold;
+  white-space: nowrap;
+}
+
+.j40FooterLogo {
+  max-width: 120px;
```

Comment:
I think a [width class](https://designsystem.digital.gov/utilities/height-and-width/) might be more resilient to resizing

Reply:
Apparently `.usa-footer__logo-img` has `max-width: 5rem;` already actually [has](https://github.com/uswds/uswds/blob/develop/src/stylesheets/components/_footer.scss) `max-width: units(10);`